tests/fish_user_function/refresh.fish

```diff
@@ -0,0 +1,14 @@
+# SYNOPSIS
+#   refresh
+#
+# OVERVIEW
+#   Refresh (reload) the current fish session.
+
+function refresh -d "refresh the fish session"
+  if set -q CI
```

Comment:
Do we need this check? Don't we ensure that `tests/fish_user_function` is loaded only on CI environment?